### PR TITLE
Dashboard Updates.

### PR DESCRIPTION
**Pull request by skmshaffer**
Makes Dashboard Updates from Unresolved Comments on Dashboard PR:
- Updates all UI elements on RSO Card to be Black. 
- Updates App.js to maintain global state for Notifications
- Updates Notifications with 'X' for closing notifications
- Runs Prettier on Codebase

Annotation 2019-10-23 004501

Closes #35

### CODE REVIEW

File: src/pages/Reimbursement.js [hunk 1]
```diff
@@ -1,150 +1,244 @@
...
+  handleSubmit(event) {
+    this.validateInput();
+    console.log(this.state);
+  }
 
-	isNotEmpty(val) {
-		return val != null && val != undefined && val != '';
-	}
+  isNotEmpty(val) {
+    return val != null && val != undefined && val != "";
```
Comment (jSith) [inline src/pages/Reimbursement.js:74]: Prefer !== here over !=

File: src/pages/Reimbursement.js [hunk 2]
```diff
@@ -1,150 +1,244 @@
...
+              invalid={!this.state.payToValid}
+            />
+            <FormFeedback invalid="true">
+              Please provide the recipient of the reimbursement.
+            </FormFeedback>
+          </FormGroup>
+          <FormGroup>
+            <Label for="amount">Amount:</Label>
+            <Input
+              type="text"
```
Comment (jSith) [inline src/pages/Reimbursement.js:182]: Why not use type="number" here rather than text?

File: src/pages/Dashboard.js [hunk 3]
```diff
@@ -38,15 +36,20 @@ const Dashboard = () => {
           </div>
         </Col>
         <Col lg="4">
-          <h6>NOTIFICATIONS ({notifications.length})</h6>
+          <h6>NOTIFICATIONS ({props.notifications.length})</h6>
           <div className="mt-3">
-            {notifications.map((notification, i) => {
+            {props.notifications.map((notification, i) => {
```
Comment (jSith) [inline src/pages/Dashboard.js]: Total nitpick, but if you're not using the index i, then I would remove it

**Review by jSith — CHANGES_REQUESTED** [review]

File: src/pages/Reimbursement.js [hunk 4]
```diff
@@ -1,150 +1,244 @@
...
+export default class ReimbursementForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      event: "",
+      description: "",
+      payTo: "",
+      amount: "",
+      file: "",
+      mail: true,
```
Comment (jSith) [inline src/pages/Reimbursement.js:23]: Sorry if I'm missing something here -- you have mail as a boolean here, but appear to be treating it like a string "on"/"off" later on? I'm a little confused on what's happening.

For the record I would prefer the boolean approach

File: src/pages/Dashboard.js (hunk 3 above)
Comment (allisoninman) [reply src/pages/Dashboard.js]: Will fix in a separate PR

File: src/pages/Reimbursement.js (hunk 4 above)
Comment (allisoninman) [reply src/pages/Reimbursement.js:23]: For some reason the radio buttons are weird. Starting it with on/off doesn't work, but updating it later with booleans doesn't work. This lets one of them start checked.

File: src/pages/Reimbursement.js (hunk 1 above)
Comment (allisoninman) [reply src/pages/Reimbursement.js:74]: Will fix in a separate PR

File: src/pages/Reimbursement.js (hunk 2 above)
Comment (allisoninman) [reply src/pages/Reimbursement.js:182]: In the validation, I allow an optional $ before the number. We can remove this if we want

Comment (jSith) [reply src/pages/Reimbursement.js:182]: I would prefer to use type number? And then if we want to show a $ we can use an addon

**Pull request merged**